app/utils/remix.server.test.ts

```diff
@@ -0,0 +1,41 @@
+import { format, parse } from '@tusbar/cache-control'
+import { describe, expect, test } from 'vitest'
+import { getConservativeCacheControl } from './remix.server.ts'
+
+describe('getConservativeCacheControl', () => {
```

Comment:
I prefer to not use `describe` because it's unnecessary. Just put the test blocks at the root of the file.